yarn.lock

```diff
@@ -456,7 +456,7 @@
 
 "@babel/runtime@^7.17.2":
   version "7.19.0"
-  resolved "https://registry.yarnpkg.com/@babel/runtime/-/runtime-7.19.0.tgz#22b11c037b094d27a8a2504ea4dcff00f50e2259"
```

Comment:
My environment seems to have changed the registry. Can you please share your setup so that we don't get this diff and use the registry you intended?

Let's add this setup to the docs for other contributors.